src/content/docs/apm/agents/php-agent/other-integrations/magento-integration.mdx

```diff
@@ -0,0 +1,257 @@
+                        - name: MYSQLConversionRate
+                        run: WITH cte1 AS (SELECT soi.product_id AS 'SalesProductId', soi.name AS 'ProductName', COUNT(soi.order_id) AS 'totalOrders' FROM `sales_order_item` soi group by soi.product_id, soi.name), cte2 AS (SELECT viewed.product_id AS 'productId', COUNT(viewed.product_id) AS 'totalViews' FROM `report_viewed_product_index` viewed group by viewed.product_id) SELECT SalesProductId as 'productID', ProductName, totalViews,totalOrders,totalOrders/totalViews *100 as 'conversionRateInPercent' FROM cte1 JOIN cte2 WHERE cte1.SalesProductId = cte2.ProductId;
+                        - name: MYSQLTopTenSalesOrdersByValue
+                        run: SELECT entity_id AS 'orderId', total_item_count AS 'orderQuantity', created_at AS 'createdDate', grand_total AS 'grandTotal' FROM sales_order ORDER BY grand_total DESC LIMIT 10;
+                        - name: MYSQLOrdersAverage
+                        run: SELECT AVG(order_count) AS 'AverageNumberOfOrders', Year AS 'Year' FROM (SELECT YEAR(created_at) AS 'Year', COUNT(DISTINCT order_id) AS order_count FROM sales_order_item GROUP BY MONTH(created_at), YEAR(created_at)) subquery GROUP BY Year;
+                        - name: MYSQLProductsTotal
+                        run: SELECT count(*) AS 'Total Products' FROM catalog_product_entity_varchar WHERE attribute_id = 73;
+                        custom_attributes: # can apply additional at a nested level
+                            attype: blocking-query
+                            host: CustomMagentoMySQL
+
+
+                ```
+
+  
+        3. Restart the infrastructure agent
+
+        To start sending all the Flex query results to [one.newrelic.com](one.newrelic.com), you'll need to restart the infrastructure agent. Follow the steps on [Start, stop, and restart the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/).
+
+        Once you've got Flex queries in New Relic, you can use a NRQL query to view data and create dashboards and charts based on queries. 
+        
+        For example, query the following to view total registered customers on your magento platform:
+
+        ```
+        SELECT LATEST(`Registered Accounts`) AS ‘Registered Accounts’ FROM MYSQLCustomerEntityRegisteredUsers
+        ```
+
+    </Step>
+
+<Step>
+    ## View your app data
+
+    To instrument the Magento quickstart and to see metrics and alerts, follow the steps on the [Magento Business Insights quickstart page](https://newrelic.com/instant-observability/magento).
+
+    To view your Magento app data, add our pre-built dashboard named “Magento Business Insights” to your account: 
+        1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** and search for "Magento". You should see our pre-build dashboard named “Magento Business Insights”. Click on it to add it to your account.
```

Comment:
On Line 21, we say "After setting up our Magento integration, we give you a dashboard for your Magento app data." Is this only available once you've set up all the agents/integrations? Or do you need to follow the instructions on the quickstart page, which includes adding the dashboard to your account? If this is true, then I'll make this a mandatory step, where you need to follow the quickstart page to add the dashboard to view your Magento data.

The info about the Magento quickstart is also confusing.

Reply:
The title should be 'Add the Magento Business Insights dashboard'